examples/launch/launch-sweeps/custom-scheduler/custom_job_sweep_config.yaml

```diff
@@ -0,0 +1,11 @@
+# custom_job_sweep_config.yaml
+scheduler:
+  job: '<entity/proj/ JOB-NAME: alias>'
```

Comment:
<entity>/<project>/<job-name>:<alias>

Reply:
This is the "custom job sweep config" so its impossible to have a pre-est scheduler job. The whole point is that the user makes one and inputs it in...